lib/exchange/authorizationCode.js

```diff
@@ -6,6 +6,8 @@
 /**
  * Module dependencies.
  */
+var SG = require('strong-globalize');
+var g = SG();
```

Comment:
We can shorten these to `var g = require('strong-globalize');` See https://github.com/strongloop-internal/loopback-knowledge-base/pull/51/files

I'll leave it up to you whether you want to address it though since these are nitpicks anyways.